internal/fs/fs_test.go

```diff
@@ -752,9 +752,6 @@ func TestIsDir(t *testing.T) {
 			if want.exists != got {
 				t.Fatalf("expected %t for %s, got %t", want.exists, f, got)
 			}
-			if !want.err {
-				t.Fatalf("expected no error, got %v", err)
-			}
 		}
 
 		if got != want.exists {
```

Comment:
Woops, not sure why I checked that in. Added that code back and altered an incorrect test expectation a few lines above.